data/json/items/vehicle_parts.json

```diff
@@ -140,6 +140,21 @@
     "category": "veh_parts",
     "price": 90000
   },
+  {
+    "type": "GENERIC",
+    "id": "recharge_station_air",
+    "name": "air pumping station",
+    "description": "A universal recharging station designed to operate on vehicle power.  While on it will steadily fill all refillable high pressure air containers placed directly within its storage space.  The system can only be installed onto existing storage compartments, and is controlled from a dashboard or electronics control unit.",
```

Comment:
```suggestion
    "description": "A universal air pumping station designed to operate on vehicle power.  While on it will steadily fill all refillable high pressure air containers placed directly within its storage space.  The system can only be installed onto existing storage compartments, and is controlled from a dashboard or electronics control unit.",
```